apps/Gruntfile.js

```diff
@@ -359,53 +359,53 @@ module.exports = function (grunt) {
+    'levels/_contract_match':       './src/sites/studio/pages/levels/_contract_match.jsx',
     'levels/dashboardDialogHelper': './src/sites/studio/pages/levels/dashboardDialogHelper.js',
-    'levels/external':              './src/sites/studio/pages/levels/external.js',
-    'levels/levelGroup':            './src/sites/studio/pages/levels/levelGroup.js',
+    'levels/_external':             './src/sites/studio/pages/levels/_external.js',
+    'levels/_level_group':          './src/sites/studio/pages/levels/_level_group.js',
     'levels/multi':                 './src/sites/studio/pages/levels/multi.js',
-    'levels/textMatch':             './src/sites/studio/pages/levels/textMatch.js',
-    'levels/widget':                './src/sites/studio/pages/levels/widget.js',
-    'signup':                       './src/sites/studio/pages/signup.js',
-    'raceInterstitial':             './src/sites/studio/pages/raceInterstitial.js',
-    'termsInterstitial':            './src/sites/studio/pages/termsInterstitial.js',
-    'makerlab/setupPage':           './src/sites/studio/pages/setupMakerlab.js',
-    'scriptOverview':               './src/sites/studio/pages/scriptOverview.js'
+    'levels/_text_match':           './src/sites/studio/pages/levels/_text_match.js',
+    'levels/_widget':               './src/sites/studio/pages/levels/_widget.js',
+    'devise/registrations/new':     './src/sites/studio/pages/devise/registrations/new.js',
+    'layouts/_race_interstitial':   './src/sites/studio/pages/layouts/_race_interstitial.js',
+    'layouts/_terms_interstitial':  './src/sites/studio/pages/layouts/_terms_interstitial.js',
+    'maker/setup':                  './src/sites/studio/pages/maker/setup.js',
+    'scripts/show':                 './src/sites/studio/pages/scripts/show.js'
   };
 
   var otherEntries = {
-    plc: './src/sites/studio/pages/plc.js',
+    'peer_reviews/show': './src/sites/studio/pages/peer_reviews/show.js',
 
     // Build embedVideo.js in its own step (skipping factor-bundle) so that
     // we don't have to include the large code-studio-common file in the
     // embedded video page, keeping it fairly lightweight.
     // (I wonder how much more we could slim it down by removing jQuery!)
     // @see embed.html.haml
-    embedVideo: './src/sites/studio/pages/embedVideo.js',
+    'videos/embed': './src/sites/studio/pages/videos/embed.js',
 
     // embedBlocks.js is just React, the babel-polyfill, and a few other dependencies
     // in a bundle to minimize the amount of stuff we need when loading blocks
     // in an iframe.
-    embedBlocks: './src/sites/studio/pages/embedBlocks.js',
+    'levels/embed_blocks': './src/sites/studio/pages/levels/embed_blocks.js',
 
     // tutorialExplorer for code.org/learn 2016 edition.
-    tutorialExplorer: './src/tutorialExplorer/tutorialExplorer.js',
+    'learn/index': './src/sites/code.org/pages/learn/index.js',
```

Comment:
this is awkward because both http://code.org/learn/ and http://hourofcode.com/learn/ load `learn/index.js`, even though they have different `learn/index.haml` files.